test/e2e/framework/prometheus.go

```diff
@@ -147,7 +147,7 @@ func (f Framework) PrometheusConfigFromSecret(t *testing.T, namespace, secretNam
 	require.NoError(t, err)
 	unzippedData, err := io.ReadAll(reader)
 	require.NoError(t, err)
-	prometheusConfig, err := promConfig.Load(string(unzippedData), false, log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr)))
+	prometheusConfig, err := promConfig.Load(string(unzippedData), promslog.NewNopLogger())
```

Comment:
Can we use [New()](https://pkg.go.dev/github.com/prometheus/common/promslog#New) here instead? That should preserve earlier behavior of logging to `Stderr`.

Reply:
I added a comment about this that apparently disappeared:
"I don't think that's worth the complexity, we can always add it in the future if needed for debugging"

Reply:
But, it's actually not complex, I'll take care of that.